setup.py

```diff
@@ -7,7 +7,7 @@
     from distutils.core import setup
 
 requirements = [
-    'Django==2.1.11',
+    'Django==2.2',
```

Comment:
is it on purpose? it doesn't cause any incompatibility with the django extensions we are using?